_pages/home.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: splash
+# title: "Mi-V Ecosystem"
+permalink: /
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+ #actions: "#5e616c"
+  #  - label: "<i class='fas fa-download'></i> Install now"
+   #   url: "/docs/quick-start-guide/"
+excerpt: >
+  Welcome to the Mi-V Ecosystem webpage.<br />
+  A comprehensive offering to ease the adoption of RISC-V.<br /> 
+  Open. Lowest power. Programmable RISC-V solutions.<br />
+feature_row:
+  - image_path: /assets/images/200602-FPGA-SOC-PolarFire-SoC-Marketo-300x500.jpg
+    alt: "customizable"
+    title: "PolarFire SoC"
+    excerpt: "They're the hard processors."
+    url:  "/docs/polarfire-soc/"
+    btn_class: "btn--primary"
+    btn_label: "Getting Started"
+  - image_path: /assets/images/190731-FPGA-Risc-v-webinar-300x500.jpg
+    alt: "fully responsive"
+    title: "Mi-V RISC-V Processors"
+    excerpt: "They're the soft processors"
+    url: "/docs/mi-v-soft-cpu/"
+    btn_class: "btn--primary"
+    btn_label: "Getting Started"
+  - image_path: /assets/images/Toolbox_iStock_300x500.png
```

Comment:
Just a suggestion...
Out of the three images on the page, only this image doesnt have the background color and border... it might look better with similar background color/border.

Reply:
Will update this image next week, Ok to merge to main for now.